lib/bolt/cli.rb

```diff
@@ -314,6 +314,11 @@ def validate(options)
         raise Bolt::CLIError, "Must specify a module name."
       end
 
+      if options[:subcommand] == 'module' && options[:action] == 'install' && options[:object]
```

Comment:
We may want to join this with `options[:leftovers]` as well, so that `bolt module install puppetlabs-apache puppetlabs-yaml` raises the same error and doesn't raise `Unknown argument(s)`

Reply:
I'm just gonna move that check for `:leftovers` to happen _after_ the check for `:object`. Seems like the correct logical ordering anyways.